### PR TITLE
You can now suicide with handcuffs!

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: You can now suicide with handcuffs!
/:cl:

### CODE REVIEW

**Comment by Buggy123** [comment]
>best idea for handcuff suicide is swallowing

P A T H E T I C

**Comment by MoreRobustThanYou** [comment]
@Buggy123 shhh you eat your hands too

File: code/game/objects/items/handcuffs.dm [hunk 1]
```diff
@@ -33,9 +33,15 @@
 	materials = list(MAT_METAL=500)
 	breakouttime = 600 //Deciseconds = 60s = 1 minute
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 50)
+	can_suicide = TRUE
 	var/cuffsound = 'sound/weapons/handcuffs.ogg'
 	var/trashtype = null //for disposable cuffs
 
+//pretty sure swalloing these would choke you
+/obj/item/restraints/handcuffs/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is swalloing [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Comment (Repukan) [inline code/game/objects/items/handcuffs.dm]: Swallowing

Comment (davethwave) [reply code/game/objects/items/handcuffs.dm]: how about swallowing the tongue or holding the breath?

**Comment by Buggy123** [comment]
How about something like "[user] flips into a handstand, hitting their neck with the chain as they go! It looks like they're trying to commit suicide!"

**Comment by optimumtact** [comment]
you can't suicide in cuffs to prevent salty people suiciding instead of ghosting

**Comment by duncathan** [comment]
fun fact, ghosting literally exists as an alternative to suicide while restrained, since suicide while restrained has been intentionally prevented for years and years. that's not gonna change just for a cute suicide text.

**Comment by Buggy123** [comment]
Why can't you suicide while cuffed? Ghosting is very nearly the same except for a handful of very specific circumstances.

**Comment by optimumtact** [comment]
It is a slap in the teeth to security for when they cuff someone.

ghosting is the cowards way out and therefore insults the ghostee more

**Comment by PKPenguin321** [comment]
if you need to keep somebody alive for whatever reason, cuffs can prevent them from just offing themselves

allows for hostage situations and stuff

**Comment by FantasticFwoosh** [comment]
Isn't @MoreRobustThanYou just digging up threads that have already been closed without research?

**Pull request closed without merging**